packages/frontend/src/components/Target.vue

```diff
@@ -74,24 +73,33 @@ export default {
     orgAndSpaceSetVisibility() {
       return this.areOrgAndSpaceSet || (this.target.currentOrg && this.target.currentSpace) ? "" : "none";
     },
+    statusApplyButton() {
+      return (
+        this.selectedOrg.label === undefined ||
+        this.selectedSpace.label === undefined ||
+        (this.selectedOrg.label === this.currentOrg && this.selectedSpace.label === this.currentSpace)
+      );
```

Comment:
the endpoint value is not counted here. in case it empty the apply allowance does not affected. isn't it ?

Reply:
the statusApplyButton button is related to org and space, not to the endpoint, user and password.. we want this button to be enabled just when the org and space fields are filled, and not the same ones as the current target.